9.5/README.md

```diff
@@ -0,0 +1,84 @@
+Users can choose between RHEL and CentOS based images.
+
+
+Environment variables and volumes
+----------------------------------
+
+The image recognizes the following environment variables that you can set during
+initialization by passing `-e VAR=VALUE` to the Docker run command.
+
+|    Variable name             |    Description                                 |
+| :--------------------------- | ---------------------------------------------- |
+|  `POSTGRESQL_USER`           | User name for PostgreSQL account to be created |
+|  `POSTGRESQL_PASSWORD`       | Password for the user account                  |
+|  `POSTGRESQL_DATABASE`       | Database name                                  |
+|  `POSTGRESQL_ADMIN_PASSWORD` | Password for the `postgres` admin account (optional)     |
+
+The following environment variables influence the PostgreSQL configuration file. They are all optional.
+
+|    Variable name              |    Description                                                          |    Default
+| :---------------------------- | ----------------------------------------------------------------------- | -------------------------------
+|  `POSTGRESQL_MAX_CONNECTIONS` | The maximum number of client connections allowed. This also sets the maximum number of prepared transactions. |  100
+|  `POSTGRESQL_SHARED_BUFFERS`  | Sets how much memory is dedicated to PostgreSQL to use for caching data |  32M
+
+You can also set the following mount points by passing the `-v /host:/container` flag to Docker.
+
+|  Volume mount point      | Description                           |
+| :----------------------- | ------------------------------------- |
+|  `/var/lib/pgsql/data`   | PostgreSQL database cluster directory |
+
+**Notice: When mouting a directory from the host into the container, ensure that the mounted
+directory has the appropriate permissions and that the owner and group of the directory
+matches the user UID or name which is running inside the container.**
+
+Usage
+----------------------
+
+For this, we will assume that you are using the `centos/postgresql-94-centos7` image.
+If you want to set only the mandatory environment variables and not store the database
+in a host directory, execute the following command:
+
+```
+$ docker run -d --name postgresql_database -e POSTGRESQL_USER=user -e POSTGRESQL_PASSWORD=pass -e POSTGRESQL_DATABASE=db -p 5432:5432 centos/postgresql-94-centos7
+```
+
+This will create a container named `postgresql_database` running PostgreSQL with
+database `db` and user with credentials `user:pass`. Port 5432 will be exposed
+and mapped to the host. If you want your database to be persistent across container
+executions, also add a `-v /host/db/path:/var/lib/pgsql/data` argument. This will be
+the PostgreSQL database cluster directory.
+
+If the database cluster directory is not initialized, the entrypoint script will
+first run [`initdb`](http://www.postgresql.org/docs/9.4/static/app-initdb.html)
```

Comment:
This link may be changed to http://www.postgresql.org/docs/9.5/static/app-initdb.html